pygeoapi/api.py

```diff
@@ -586,6 +587,16 @@ def describe_processes(self, headers, args, process=None):
 
         headers_ = HEADERS.copy()
 
+        format_ = check_format(args, headers)
```

Comment:
AFAIKS `check_format()` may return `None` in cases (e.g. when format other than `json` or `html` is requested, try e.g. `?f=gml`).
The code below must handle that case, returning exception, otherwise `json` is assumed.

Reply:
Hmm, rethinking, makes sense: by default JSON is returned, even if no format specified (e.g. when using curl), so forget above comment.